package/android/cpp/rnskia-android/SkiaOpenGLRenderer.h

```diff
@@ -26,96 +28,44 @@
 #pragma clang diagnostic pop
 
 namespace RNSkia {
-sk_sp<SkSurface> MakeOffscreenGLSurface(int width, int height);
-
-using OpenGLDrawingContext = struct {
-  EGLContext glContext;
-  EGLDisplay glDisplay;
-  EGLConfig glConfig;
-  sk_sp<GrDirectContext> skContext;
-};
 
-static std::unordered_map<std::thread::id,
-                          std::shared_ptr<OpenGLDrawingContext>>
-    threadContexts;
+/**
+ * The first context created will be considered the parent / shared context and
+ * will be used as the parent / shareable context when creating subsequent
+ * contexts.
+ */
+static EGLContext sharedEglContext = EGL_NO_CONTEXT;
```

Comment:
The only way I was able to create a root shared context is by creating an offscreen 1x1 surface: https://github.com/Shopify/react-native-skia/pull/1659/files#diff-d366ce631dd3731257cb2c1f7123a6f932d7e01c87952d1f2cb93eb881c6d8e7R77

The same pattern can be found in skottielib: https://github.com/google/skia/blob/main/platform_tools/android/apps/skottie/skottielib/src/main/java/org/skia/skottie/SkottieRunner.java#L201